api-mock.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import m "github.com/lsolniczek/api-mock/api-mock-main"
```

Comment:
This is kind of awkward that `main`-specific logic is not in the `main` package. What's the reasoning here?